vertical-pod-autoscaler/pkg/utils/vpa/capping.go

```diff
@@ -30,88 +30,127 @@ func NewCappingRecommendationProcessor() RecommendationProcessor {
 	return &cappingRecommendationProcessor{}
 }
 
+type cappingAction = string
+
+var (
+	cappedToMinAllowed cappingAction = "capped to min allowed"
+	cappedToMaxAllowed cappingAction = "capped to max allowed"
+	cappedToLimit      cappingAction = "capped to limit"
```

Comment:
capped to request limit? Not very sure though, limit might be specific enough

Reply:
Changed to "capped to container limit".